tests/unit/plugins/lookup/test_onepassword.py

```diff
@@ -82,6 +82,15 @@ def test_assert_logged_in_v2(mocker, args, out, expected_call_args, expected_cal
     assert result == expected
 
 
+def test_assert_logged_in_v2_connect(mocker, monkeypatch):
+    mocker.patch.object(OnePassCLIv2, "_run", return_value=[1, "", ""])
+
+    op_cli = OnePassCLIv2(connect_host="http://localhost:8080", connect_token="foobar")
+    result = op_cli.assert_logged_in()
+    assert not op_cli._run.called
+    assert result
```

Comment:
This test can be simplified.
```suggestion
def test_assert_logged_in_v2_connect():
    op_cli = OnePassCLIv2(connect_host="http://localhost:8080", connect_token="foobar")
    result = op_cli.assert_logged_in()

    assert result is True
```

Reply:
We also need more tests for the changed code.